src/ui/MobileMenu/MobileBottomSheet.tsx

```diff
@@ -101,6 +101,7 @@ const MobileBottomSheet: React.FunctionComponent<MobileBottomSheetProps> = (prop
                           hideMenu();
                           toggleReaction(message, emoji.key, isReacted);
                         }}
+                        dataSbId={emoji.key}
```

Comment:
something like
```
dataSbId={`emoji_bottom-sheet_${emoji.key}`}
```
would be more trackable

Reply:
okay okay let me apply it